spec/support/helpers/booking_response_helper.rb

```diff
@@ -1,5 +1,5 @@
 module BookingResponseHelper
-  def reject_visit(visit, reasons = [Rejection::NO_ALLOWANCE])
+  def reject_visit(visit, reasons = [Rejection::SLOT_UNAVAILABLE])
```

Comment:
Why the default value for reasons? I wouldn't expect a default value to be necessary as the reasons should always be explicitly set